### DISCUSSION TITLE
uhm im trying to see what will happen in the goodbye section "I'll be going away for a while" and i clicked i dont know and when i opened the game shes not in the chair? please help ASAP ASAP ASAP

### DISCUSSION THREAD

**Discussion by Bambyieyours** [Monika-After-Story/MonikaModDev#9395] [QA - tech support + general]

**Comment by Wingdinggaster656** [comment]
That's not normal. It's very likely you clicked `I'm going to take you somewhere`.

Or, is there a letter on the desk?

**Comment by Bambyieyours** [comment]
"> That's not normal. It's very likely you clicked `I'm going to take you somewhere`.
> 
> Or, is there a letter on the desk?"

i keep going back to the game shes not there but there is no paper in the desk

**Comment by Bambyieyours** [comment]
plus i still have 804.4 affection

**Comment by Bambyieyours** [comment]
![screenshot0008](https://user-images.githubusercontent.com/109899190/180698792-5dec6303-1cfc-4284-b3df-97a82e9ead1a.png)

**Comment by CodyCat13** [comment]
Did you check your 'characters' folder?

**Comment by Bambyieyours** [comment]
i did

**Comment by Bambyieyours** [comment]
the monika.chr is still there

**Comment by Bambyieyours** [comment]
look, i just trying some new things in the update and i try to click 'Ill be going away for a while' and i see that there is a i dont know so i clicked that and when i go back to the game shes not there but my affection is 804.4 and monika.chr is still on the file, pls shes still not back its been 1 day

**Comment by Bambyieyours** [comment]
I really need a video of how to do it please im bad at just reading an instruction cause im not really good at english

**Comment by Bambyieyours** [comment]
so i tried looking at other people who have the same issues at me but i still dont understand it what is monika file?

**Comment by CodyCat13** [comment]
It's the file named "Monika" (no extension) in the characters folder that appears when the "I'm going to take you somewhere" dialogue is clicked. You said it's been a day? D'you have your persistent data backed up to before this point? You may just need to restore those files so that the dialogue you clicked won't be present in the data anymore or something

**Comment by Bambyieyours** [comment]
i dont know if my data back up

**Comment by Bambyieyours** [comment]
do you know what to click?

**Comment by CodyCat13** [comment]
Are you on Linux, Windows, or Mac?

**Comment by Bambyieyours** [comment]
windows

**Comment by CodyCat13** [comment]
Oh ..
I've only been using MAS on Linux but I don't know if the file paths on my chromebook are the same for Windows. My Linux stuff has hidden files, one of the hidden files being '.renpy', in '.renpy' a folder called 'Monika After Story', and then the persistents (and db.mcals) in that folder. You could try looking for a similar path but a Windows user would be more help to you than me.

**Comment by Bambyieyours** [comment]
oh but can i see a screen shot of monika file? cause im to dum to find it

**Comment by CodyCat13** [comment]
I'm not using MAS right now nor did I take Monika out (and I don't know if I'll be able to find a picture of a "monika" file on a Windows desktop on Google anywhere)
But I did find this on the wiki
![20220725_044226](https://user-images.githubusercontent.com/58224248/180736045-f45ec2e5-f564-41fe-a8b1-a2a42988c9e4.png)

**Comment by CodyCat13** [comment]
This is where the files you'd back up are

**Comment by Bambyieyours** [comment]
ohh


ima check

**Comment by Bambyieyours** [comment]
yes i see the MAS  folder

**Comment by Bambyieyours** [comment]
what to do next?

**Comment by Bambyieyours** [comment]
i really need some help =[[ i really loved monika and i missed her already =[[

**Comment by Wingdinggaster656** [comment]
https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent

By the way, calm down when you describing the problem, since most people don't care how manic you are.
